apps/ui-patterns/src/app/App.tsx

```diff
@@ -1,13 +1,13 @@
 import React from 'react';
-import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 import UIPatterns from './UIPatterns';
 
 const App: React.FC = () => {
   return (
     <Router>
-      <Routes>
-        <Route path="/" element={<UIPatterns />} />
-      </Routes>
+      <Switch>
+        <Route path="/" component={UIPatterns} />
+      </Switch>
```

Comment:
Hmm, we may have a version mismatch on React, here. Latest react-router-dom uses Routes instead of Switch

Reply:
We would have, but... in the commit of this, e859114, I also reverted the `react-router-dom` to a version near/at CEPv2. In another commit, 4a873cb, I reverted `reactstrap` to match `bootstrap@^4.6.0` that `index.css` loads.

Now that I see it working with the versions it was built on, I feel more comfortable upgrading and resolving the issues caused by changing props.

The cause of this was `activeClassName` prop **not** being in the latest version. Instead we need to set the `active` prop to a boolean (and then set active class name dependent on that new boolean). I do not yet see how to get such a Boolean.

Reply:
> Related to the older versions of react router Dom, etc, let's stick with what is compatible with cep v2 — jchuahtacc [internal comm.](https://tacc-team.slack.com/archives/C02GF2X2AS3/p1655820645562929?thread_ts=1655757404.634289&cid=C02GF2X2AS3)